### PR TITLE
*: provide SHOW PLUGINS statement with a dummy implementation

### PR DESCRIPTION
**Pull request by tiancaiamao**
For issue https://github.com/pingcap/tidb/issues/4172

@XuHuaiyu @hanfei1991 @zimulala

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -5463,12 +5464,18 @@ ShowTargetFilterable:
...
-        $$ = &ast.ShowStmt{
-        	Tp:	ast.ShowEvents,
-        	DBName:	$2.(string),
-       	}
+		$$ = &ast.ShowStmt{
+			Tp:	ast.ShowEvents,
+			DBName:	$2.(string),
+		}
+	}
+|	"PLUGINS"
```
Comment (zimulala) [inline parser/parser.y]: Do we need to add a test in `parser_test`?

Comment (tiancaiamao) [inline parser/parser.y]: Done.

**Comment by zimulala** [comment]
LGTM

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**